.circleci/config.yml

```diff
@@ -856,8 +858,9 @@ workflows:
           filters: { branches: { ignore: /^deploy\/.*/ } }
 
       # Run tests on a single container
-      - build-and-test-go:
-          filters: { branches: { ignore: /^deploy\/.*/ } }
+      # FIXME
+      # - build-and-test-go:
```

Comment:
Whats up with this?

Reply:
It's failing, also in James' PR. I'm not sure why, I'm not sure it's important since GHA tests this

Reply:
https://github.com/mobilecoinfoundation/mobilecoin/pull/2514#issuecomment-1246893946

Reply:
The error is something about not finding the test stub binary.